app/models/models.py

```diff
@@ -16,8 +16,8 @@ def to_questionjson(self):
         json representation of the questions model
         """
         return {
-            'id': self.qtn_id,
+            'id': self.Question_ID,
             'title': self.title,
-            'body': self.body,
+            'Description': self.body,
```

Comment:
Why do you use body in the model and description when returning the data. Choose a consistent naming scheme.